src/uct/ib/ud/base/ud_iface.c

```diff
@@ -738,6 +757,8 @@ static void uct_ud_iface_cq_event(void *arg)
             }
             break;
         }
+
+        ucs_debug("GET_CQ_EVENT read fd %d ret %d", iface->super.comp_channel->fd, ret);
```

Comment:
after the change, need to remove the call for ibv_get_cq_event() as well. events would be cleared by wakeup mechanism, if it would be used,